src/main/java/org/broadinstitute/consent/http/models/UserRole.java

```diff
@@ -79,7 +79,7 @@ public void setDacId(Integer dacId) {
 
     @Override
     public int hashCode() {
-        return Objects.hashCode(this.userRoleId, this.name, this.dacId);
+        return Objects.hashCode(this.userId, this.name, this.dacId);
```

Comment:
why not include both userId and userRoleId?

Reply:
I think Elliot has it right. Alternatively, `(this.userId, this. this.userRoleId, this.dacId)` should also work. Comparing role id and name together is not helpful since if I have two chair roles, both of them will have the same role id and name, they'd only be distinguishable by the dac id.

Reply:
Good question.  We could, but I think we can do a refactor to get rid of the userRoleId all together.  I'd like to consolidate and reduce all of the user_role table actions into the UserRoleDAO.